KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -878,12 +942,16 @@ private void turnSwitchBtnActionPerformed(java.awt.event.ActionEvent evt) {//GEN
             trade3to1Btn.setEnabled(false);
             trade2to1Btn.setEnabled(false);
 
+            //disable all the SettlerBtns
+            toggleCardBtn.setEnabled(false);
+            buyDevCardBtn.setEnabled(false);
+
             // Change the button to the Start Next Turn button
             turnSwitchBtn.setText("Start Player " + currentPlayer + "'s Turn");
 
             //update the instruction
             instructionLbl.setText("Please allow the next player to use the mouse");
-            subInstructionLbl.setText("");
+            subInstructionLbl.setText("Then start the next turn");
```

Comment:
fixes bug the makes the build buttons move up and down when a turn is ended or started.